MementoMori/MementoMori.Server/Controllers/AuthController.cs

```diff
@@ -13,14 +15,15 @@ public class AuthController: ControllerBase
     {
         private readonly IAuthService _authService;
         private readonly IAuthRepo _authRepo;
+        private readonly IDeckHelper _deckHelper;
         private static readonly ConcurrentDictionary<string, User> _registeredUsers = new();
         private static bool initialized = false;
         
-        public AuthController(IAuthService authService, IAuthRepo authRepo)
+        public AuthController(IAuthService authService, IAuthRepo authRepo, IDeckHelper deckHelper)
```

Comment:
Dont think authentication should depend on decks, create new controller for this.

Reply:
Done.